### DISCUSSION TITLE
Asciinema edit

### DISCUSSION THREAD

**Discussion by pocc** [asciinema/discussions#56] [Show and tell]
I made [a small tool](https://github.com/pocc/asciinema-edit) to cut frames out and rearrange sections by timestamp. Is there interest in merging it into Asciinema?

Regardless of whether this specific implementation is the "correct" solution to the problem of editing, some form of post-processing should be added to the tool. There also exists a [solution in golang](https://github.com/cirocosta/asciinema-edit) by @cirocosta, which also has good features.

For the record, [this forum reply](https://discourse.asciinema.org/t/can-i-edit-post-process-the-recording/25) is misleading and SHOULD point people to existing projects that solve this problem.

**Comment by davedittrich** [comment]
I am interested in similar capabilities: Editing out mistakes; cutting/pasting in sequences from another asciicast; Rebasing the resulting time series to ensure proper playing of the results. Right now this kind of editing is really difficult and tedious by hand.

**Comment by pocc** [comment]
@sickill What are your thoughts?

**Comment by kakulukia** [comment]
"Reversing" the cast format and saving delay between updates rather than full times will help a lot!
This way edits can easily be made with text editor without the need to touch all following lines after editing one "frame".
